Makefile

```diff
@@ -191,7 +191,7 @@ delete-tag:
 docker-otelcol:
 	GOOS=linux $(MAKE) otelcol
 	cp ./bin/otelcol_linux_amd64 ./cmd/otelcol/otelcol
-	docker build -t otelcol ./cmd/otelcol/
+	docker build -t otelcol ./cmd/otelcol/ --build-arg SMART_AGENT_RELEASE_URL=https://github.com/signalfx/signalfx-agent/releases/download/$(SMART_AGENT_RELEASE)/signalfx-agent-$(subst v,,$(SMART_AGENT_RELEASE)).tar.gz
```

Comment:
nit, mind specifying the context as last arg?

Reply:
No, I don't mind.